serverLib.sml

```diff
@@ -365,13 +366,17 @@ structure GitHub = struct
     end
 end
 
+
+(* we ask for the first 100 PRs/labels below. github requires some
```

Comment:
'GitHub' and sentence capitalisation please. (I'm happy to change this myself later, but if you're committing more anyway...)

Reply:
OK